addon/session-stores/cookie.js

```diff
@@ -76,14 +76,16 @@ export default BaseStore.extend({
   */
   cookieExpirationTime: null,
 
-  _secureCookies: window.location.protocol === 'https:',
+  _secureCookies: computed(function () {
+    return window ? window.location.protocol === 'https:' : _fastbootInfo.host().includes('https:');
```

Comment:
`_fastbootInfo.host()` will be going away with https://github.com/ember-fastboot/ember-fastboot-server/pull/38

I will have to update references accordingly.